ScoutSuite/providers/gcp/rules/findings/stackdriverlogging-no-export-sinks.json

```diff
@@ -1,8 +1,8 @@
 {
     "dashboard_name": "Logging Configurations",
     "description": "No Export Sinks",
-    "path": "stackdriverlogging.sinks.id",
+    "path": "stackdriverlogging.projects.id.sinks.id",
     "conditions": [ "and",
-        [ "stackdriverlogging.sinks", "empty", "" ]
+        [ "stackdriverlogging.projects.id.sinks", "empty", "" ]
```

Comment:
How does the view handle this change in the data structure?

Reply:
Fixed!